client/layout/community-translator/invitation.jsx

```diff
@@ -25,48 +24,45 @@ export default React.createClass( {
 
 		invitationUtils.recordInvitationDisplayed();
 
-		let subComponents = {
+		const subComponents = {
 			title: this.translate( 'Translate WordPress.com as you go' ),
-			acceptButtonText: this.translate( 'Start now' ),
+			acceptButtonText: this.translate( 'Try it now!' ),
```

Comment:
We're settling now on `Try it now!` because of a very slight advantage in click-through rate.